sonarqube/content.md

```diff
@@ -42,14 +42,18 @@ We recommend creating volumes for the following directories:
 -	`/opt/sonarqube/conf`: **for Version 7.9.x only**, configuration files, such as `sonar.properties`.
 -	`/opt/sonarqube/data`: data files, such as the embedded H2 database and Elasticsearch indexes
 -	`/opt/sonarqube/logs`: contains SonarQube logs about access, web process, CE process, Elasticsearch logs
--	`/opt/sonarqube/extensions`: plugins, such as language analyzers
+-	`/opt/sonarqube/extensions`: for all 3rd party plugins 
 
 > **Warning:** You cannot use the same volumes on multiple instances of SonarQube.
 
 ## First Installation
 
 For installation instructions, see Installing the Server from the Docker Image on the [Install the Server](https://docs.sonarqube.org/latest/setup/install-server/) page.
 
+### Datacenter Installation
+
+If you want to run SonarQube in a cluster, please refer to the [Installing SonarQube from the Docker Image section](https://docs.sonarqube.org/latest/setup/install-cluster/#header-4) of the [Install the  Server as a Cluster guide](https://docs.sonarqube.org/latest/setup/install-cluster/).
```

Comment:
Since the 2 links point to the same page, I suggest we keep only one.

```suggestion
To run a cluster with SonarQube DataCenter Edition, please refer to Installing SonarQube from the Docker Image on the [Install the Server as a Cluster](https://docs.sonarqube.org/latest/setup/install-cluster/) page.
```